source/jormungandr/jormungandr/realtime_schedule/siri.py

```diff
@@ -108,6 +108,7 @@ def __init__(self, id, service_url, requestor_ref,
         self.instance = instance
         self.breaker = pybreaker.CircuitBreaker(fail_max=app.config.get('CIRCUIT_BREAKER_MAX_SIRI_FAIL', 5),
                                                 reset_timeout=app.config.get('CIRCUIT_BREAKER_SIRI_TIMEOUT_S', 60))
+        self.step = kwargs.get('step', 30)
```

Comment:
I'd name it `from_datetime_step` but it's still not super sexy...
at least a comment is welcome: "A step is applied on from_datetime to discretize calls and allow caching them"